docs/source/Howtos/Beginner-Tutorial/Part2/Beginner-Tutorial-Part2-Overview.md

```diff
@@ -1,25 +1,23 @@
-# Part 2: What we want
+# Part 2: What We Want
 
 ```{sidebar} Beginner Tutorial Parts
-- [Introduction](../Beginner-Tutorial-Overview.md)
+- [Introduction](./Beginner-Tutorial-Overview.md)
 <br>Getting set up.
-- Part 1: [What we have](../Part1/Beginner-Tutorial-Part1-Overview.md)
+- Part 1: [What We Have](Part1/Beginner-Tutorial-Part1-Overview.md)
 <br>A tour of Evennia and how to use the tools, including an introduction to Python.
-- **Part 2: [What we want](./Beginner-Tutorial-Part2-Overview.md)**
-<br>Planning our tutorial game and what to think about when planning your own in the future.
-- Part 3: [How we get there](../Part3/Beginner-Tutorial-Part3-Overview.md)
-<br>Getting down to the meat of extending Evennia to make our game
-- Part 4: [Using what we created](../Part4/Beginner-Tutorial-Part4-Overview.md)
-<br>Building a tech-demo and world content to go with our code
-- Part 5: [Showing the world](../Part5/Beginner-Tutorial-Part5-Overview.md)
-<br>Taking our new game online and let players try it out
+- *Part 2: [What We Want](Part2/Beginner-Tutorial-Part2-Overview.md)*
+<br>Planning our tutorial game and what to consider when planning your own.
+- Part 3: [How We Get There](Part3/Beginner-Tutorial-Part3-Overview.md)
+<br>Getting down to the meat of extending Evennia to make your game.
+- Part 4: [Using What We Created](Part4/Beginner-Tutorial-Part4-Overview.md)
+<br>Building a tech-demo and world content to go with our code.
+- Part 5: [Showing the World](Part5/Beginner-Tutorial-Part5-Overview.md)
+<br>Taking our new game online and letting players try it out.
 ```
 
-In Part two of the Evennia Beginner Tutorial we'll take a step back and plan out the kind of tutorial game we want to make. This is a more 'theoretical' part where we won't do any hands-on
-programming. 
+In Part Two of the Evennia Beginner Tutorial, we'll take a step back to plan the type of tutorial game that you will create. This part id more 'theoretical' in that you won't do any hands-on programming. 
```

Comment:
Nit-pick, but I think it sounds a bit 'friendlier' to use 'we' when referring to the game to make rather than 'you'. The latter seems like we are forcing something upon the reader while the former (IMO) is more that we are inviting them to a collaborative project between the reader and the documentation ... So I think we should stick to 'we' here.

There is also a typo here `id -> is` :)

Reply:
agreed. maybe i was caffeine-deficient when i was editing this, but i thought i was taking my cue from other places in the documentation where it goes back and forth between we and you. i, too, think _we_ is friendlier and, for example, preserved _we_ here where we are communicating joining with the reader in a collaboration of sorts. But, then I switched to _you_ when speaking specifically about actions the reader will take by following the instructions.

Reply:
I guess it doesn't matter all too much. :)